src/Config.php

```diff
@@ -27,40 +29,49 @@ public function get(string $key, mixed $default = null): mixed
     /**
      * @throws InvalidConfigException
      */
-    private function getByKeyPath(array $key)
+    private function getByKeyPath(array $key): mixed
     {
-        $file = sprintf('%s/%s.php', $this->directory, $key[0]);
+        $subDirectory = array_shift($key);
 
-        if (!file_exists($file)) {
+        $file = realpath("{$this->directory}/{$subDirectory}.php");
```

Comment:
I'm guessing you missed my rant from yesterday as to why I did this cursed sprintf shit, I strongly dislike double quotes.

I just updated the linter config to allow spaces around concats, this can be `$this->directory . DIRECTORY_SEPARATOR . $key[0] . '.php'` instead